polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -81,12 +81,6 @@ MLIRScanner::MLIRScanner(MLIRASTConsumer &Glob,
       LTInfo(LTInfo) {}
 
 void MLIRScanner::initUnsupportedFunctions() {
-  // FIXME: -no-mangled-function-name: cgeist:
-  // llvm/mlir/lib/IR/SymbolTable.cpp:121:
-  // mlir::SymbolTable::SymbolTable(mlir::Operation*): Assertion
-  // `symbolTableOp->hasTrait<OpTrait::SymbolTable>() && "expected operation to
-  // have SymbolTable trait"' failed.
-  unsupportedFuncs.insert("_ZNK4sycl3_V15rangeILi2EE4sizeEv");
   unsupportedFuncs.insert(
```

Comment:
This function has the same failure, can you check if it can be removed too?

Reply:
I'm getting a different error there:
```
cgeist: /home/victor/randd/dpcpp/polygeist/tools/cgeist/Lib/clang-mlir.cc:114: void checkFunctionParent(mlir::FunctionOpInterface, FunctionContext, const mlir::OwningOpRef<mlir::ModuleOp>&): Assertion `(Context != FunctionContext::Host || F->getParentOp(\
) == module.get()) && "New function must be inserted into global module"' failed.
```
That's before the LLVM lowering. We can address that in a different PR.

Reply:
It could be an error hidden from the previous error? 
Anyways, do you mind adding the `FIXME` comment before this function?